test/e2e/app-dir/next-image/next-image-proxy.test.ts

```diff
@@ -0,0 +1,90 @@
+import { join } from 'path'
+import {
+  findPort,
+  killApp,
+  nextBuild,
+  nextStart,
+} from '../../../lib/next-test-utils'
+import https from 'https'
+import httpProxy from 'http-proxy'
+import fs from 'fs'
+import webdriver from '../../../lib/next-webdriver'
+
+const appDir = join(__dirname)
+let proxyPort
+let app
+let proxyServer: https.Server
+
+describe('next-image-proxy', () => {
+  beforeAll(async () => {
+    await nextBuild(appDir)
```

Comment:
```suggestion
    await nextBuild(appDir)
```

Are we not able to use `createNext` utils here?